src/main/webapp/js/auth_check.js

```diff
@@ -0,0 +1,68 @@
+import {
+  standardizeFetchErrors,
+  makeRelativeUrlAbsolute,
+} from './fetch_handler.js';
+
+/**
+ * Creates the authentication fetch request.
+ * @return {Promise} Returns the authentication fetch request.
+ */
+function createAuthFetch() {
+  const errorFormattedRequest = standardizeFetchErrors(
+      fetch(makeRelativeUrlAbsolute('/auth')),
+      'Failed to communicate with authentication server.',
+      'Failed to verify authentication.');
+  return errorFormattedRequest
```

Comment:
is there any reason for separating these?

Reply:
For readability. Otherwise, the line wrapping gets pretty crazy, although if you think this is worth changing lmk.